DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Administrator/SearchableAdminViewModel.cs

```diff
@@ -31,6 +42,6 @@ public SearchableAdminViewModel(AdminUser adminUser)
 
         public bool IsLocked { get; set; }
 
-        public bool IsCentreManager { get; set; }
+        public bool CanShowDeactivateAdminButton { get; set; }

```

Comment:
I think it would be slightly nicer to make this a readonly property and move the logic from the constructor into the body of the property.

Reply:
I removed setter so the value cannot be changed outside this class and this change is sufficient 

I think we cannot move the body to read-only property body since we do not have private members that hold values from constructor

Reply:
We should still change this to a readonly property and move the logic from the constructor into the body of the property.